src/audio/codec_adapter/codec_adapter.c

```diff
@@ -426,9 +420,9 @@ int codec_adapter_copy(struct comp_dev *dev)
 			 ret);
 		goto db_verify;
 	}
+
 	codec_adapter_copy_from_lib_to_sink(&codec->cpd, &local_buff->stream,
```

Comment:
No need to add new line here. If you want to add this new line or remove the line below just create a new patch.